helper/resource/testing_new_import_state.go

```diff
@@ -94,23 +92,6 @@ func testStepNewImportState(t testing.T, c TestCase, wd *tftest.WorkingDir, step
 					r.Primary.ID)
 			}
 
-			// We'll try our best to find the schema for this resource type
```

Comment:
Not sure on this deletion, this was copy pasted directly from the old implementation of the acctest framework.

Reply:
We can let @kmoe chime in on this, but I think it was just to avoid the diff mismatch in import tests.